client/src/components/BoardDisplay/BoardDisplay.js

```diff
@@ -1,24 +1,41 @@
 import React from 'react'
 import Moment from 'moment'
+import { Link } from 'react-router-dom'
+
+import { makeStyles } from '@material-ui/core/styles'
+import Container  from '@material-ui/core/Container'
 import Card from '@material-ui/core/Card'
 import Typography from '@material-ui/core/Typography'
 import CardActionArea from '@material-ui/core/CardActionArea'
 import CardHeader from '@material-ui/core/CardHeader'
 import IconButton from '@material-ui/core/IconButton'
+import Fab from '@material-ui/core/Fab'
+
+
 import DeleteIcon from '@material-ui/icons/Delete'
 
-import { Link } from 'react-router-dom'
 
 import BoardContext from '../../utils/BoardContext'
 import EditBoardModal from '../../components/EditBoardModal'
+import AddBoardModal from '../../components/AddBoardModal'
+import DeleteBoardModal from '../../components/DeleteBoardModal'
 import './BoardDisplay.css'
 
+const useStyles = makeStyles(theme => ({
+  fab: {
+    position: 'fixed',
+    bottom: theme.spacing(2),
+    right: theme.spacing(2),
+  },
+}));
+
 const BoardDisplay = () => {
 
+  const classes = useStyles()
   const { boards } = React.useContext(BoardContext)
 
   return (
-    <>
+    <Container>
```

Comment:
To get the fixed FAB to work properly, it needs to be anchored into some component.